src/printapp/test/test-api.py

```diff
@@ -101,18 +101,14 @@ def test_upload_file(self):
             response = app.post('/api/upload', data=form_data)
             self.assertEqual(self._status(response), 400)
 
-    @unittest.skip("Skip test which actually prints.")
-    def test_print(self):
-        """Tests the /api/upload and /api/print endpoints.
-
-        This test is disabled by default. Enable it by commenting out the
-        decorator above.
+    def test_print_and_deletejob(self):
+        """Tests the /api/upload and /api/print and /api/deletejob/<job_id> endpoints.
         """
         with self.get_client() as app:
             self._sign_in(app)
 
             oid = []
-            file_names = ['test.pdf', 'test.txt', 'test.doc', 'test.docx']
```

Comment:
Yah, my guess is cloud print is behaving differently than before.

Try hitting the api a few different ways and looking at the response in
fiddler.

Check the cloudprint API documentation to see if anything has changed or if
there's a new better way to do things.
On May 6, 2015 7:16 PM, "Tyler VanZanten" <notifications@github.com> wrote:

> In src/printapp/test/test-api.py
> <https://github.com/jglamine/calvinwebprint/pull/45#discussion_r29812317>:
>
> >          """
> >          with self.get_client() as app:
> >              self._sign_in(app)
> >
> >              oid = []
> > -            file_names = ['test.pdf', 'test.txt', 'test.doc', 'test.docx']
>
> Even with test.pdf not included in the files being tested, I'm still
> having trouble now getting this test to pass.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jglamine/calvinwebprint/pull/45/files#r29812317>.
>